src/chemistry/geoschem/chemistry.F90

```diff
@@ -1391,6 +1398,21 @@ subroutine chem_init(phys_state, pbuf2d)
        CALL Print_TaggedDiagList( Input_Opt%amIRoot, TaggedDiag_List, RC )
     ENDIF
 
+    ! There are actually two copies of the history configuration, one is contained
```

Comment:
It would be helpful that the reason for this elaborated here in the comments.

Reply:
Added